app/assets/stylesheets/uswds-settings.scss

```diff
@@ -2,23 +2,51 @@
   // Point the asset pipline to the correct locations
   $theme-font-path: "@uswds/uswds/dist/fonts",
   $theme-image-path: "@uswds/uswds/dist/img",
-
   $theme-show-notifications: false,
 
-  // Put your USWDS Theme settings here
-  $output-these-utilities:
-    (
-      "background-color",
-      "color",
-      "display",
-      "font",
-      "font-style",
-      "line-height",
-      "margin",
-      "margin-horizontal",
-      "margin-vertical",
-      "padding",
-      "text-align",
-      "text-transform"
-    )
+  $theme-global-content-styles: true,
+
+  // Sizing
+  $theme-footer-max-width: "desktop-lg",
+  $theme-grid-container-max-width: "desktop-lg",
+  $theme-header-max-width: "desktop-lg",
+
+  // Colors
+  $theme-color-accent-warm-dark: "gold-50",
+  $theme-color-primary: "cyan-50",
+  $theme-color-primary-darker: "cyan-70",
+  $theme-color-primary-dark: "mint-cool-60",
+  $theme-banner-link-color: "blue-60v",
+  $theme-text-color: "gray-cool-80",
```

Comment:
[question] there seems to be a lot of color contrast warnings coming out of the build. Is that something to be worried about, or can we ignore that as long as pa11y is passing the color contrast?

Reply:
Yeah, let me see if I can fix some of those